### ISSUE TITLE
Update Chat workshop

### ISSUE THREAD

**Issue by MaxWofford** [hackclub/hackclub#912]
The chat workshop needs to be reviewed. First step is to give it a standard heading:

```md
# Chat workshop

| What you'll build       |
| ----------------------- |
| ![](img/final_demo.gif) |

Here's a link to the [live demo][live_demo] and the [final code][final_code]. This workshop should take around 1 hour.

_**We recommend going through this workshop in Google Chrome.**_

[live_demo]: https://prophetorpheus.github.io/chat
[final_code]: https://github.com/prophetorpheus/prophetorpheus.github.io/tree/master/chat

```

Next is to review the rest of the workshop and submit a PR to fix any of the bugs or bad explanations it has.

**Comment by MaxWofford** [comment]
@aarohmankad: can you take this issue?

**Comment by aarohmankad** [comment]
@MaxWofford for sure

**Comment by MaxWofford** [comment]
This issue also entails finishing up part 5 of the workshop. Check out workshops.hackclub.com/personal_website for an example "hacking" section.

**Comment by zachlatta** [comment]
Closed by #920.
